lib/valueflow.cpp

```diff
@@ -2920,6 +2920,18 @@ static bool valueFlowForwardVariable(Token* const startToken,
                 return false;
         }
 
+        // Function call
+        else if (!var->isLocal() && tok2->function() && Token::Match(tok2, "%name% (")) {
```

Comment:
This should call `exprDependsOnThis` to find out if this is a member function.

Reply:
Done